src/main/scala/org/broadinstitute/dsde/vault/model/Properties.scala

```diff
@@ -0,0 +1,8 @@
+package org.broadinstitute.dsde.vault.model
+
+object Properties {
+  val created_by = "created_by"
+  val created_date = "created_date"
+  val modified_by = "modified_by"
+  val modified_date = "modified_date"
```

Comment:
Same updates as the [DM](https://github.com/broadinstitute/vault-datamanagement/pull/26#discussion-diff-29345694) version.